constants.go

```diff
@@ -446,6 +446,8 @@ const (
 	CertExtensionPermitPTY = "permit-pty"
 	// CertExtensionPermitPortForwarding allows user to request port forwarding
 	CertExtensionPermitPortForwarding = "permit-port-forwarding"
+	// CertExtensionSSHPortForwardMode allows user to request a specific port forwarding mode
+	CertExtensionSSHPortForwardMode = "ssh-port-forward-mode"
```

Comment:
the permit-port-forwarding extension is used by OpenSSH clients and servers. Since they handle it as a binary on/off, I think we should only set it when both server and client port forwarding are allowed. I don't think we need the new ssh-port-forward-mode extension if we aren't using it